src/locale/locales/fi/messages.po

```diff
@@ -1208,7 +1208,7 @@ msgstr "Muokkaa ulkoisten sivustojen mediasisältöjen asetuksia"
 
 #: src/view/screens/Settings.tsx:687
 #~ msgid "Danger Zone"
-#~ msgstr ""
+#~ msgstr "Vaaravyöhyke"
```

Comment:
Just a friendly tip, you don't need to bother translating any of the lines starting with `#~` because this means that the string is obsolete and no longer appears in the app (you can delete these in the file if you wish, or just leave them) :)

Reply:
Yes, thank you. 🙂 I knew about the # thing, but I thought that if there's a small chance they might be reinstated at some point, it's good to have them translated.

Why aren't they removed completely?

Reply:
If thats my job, ill start cleaning 😁

Reply:
> Why aren't they removed completely?

They don't get removed automatically because sometimes there might just be a tiny change in the English string, like adding a punctuation mark, for example. So if they were removed automatically it would be less convenient for translators because they would either have to redo the translation from scratch for something where the meaning might not actually have changed, or else look up the old version of the file.